modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/individual_controller.rb

```diff
@@ -30,8 +30,11 @@ def validate
         private
 
         def validate_individual_poa_code!(poa_code)
+
           return if ::Veteran::Service::Representative.where('? = ANY(poa_codes)', poa_code).any?
 
+          return if ::Veteran::Service::Organization.find_by(poa: poa_code).blank?
```

Comment:
I missed this in the earlier review. I think both checks need to always execute instead of early returning after just the representative check.